src/main/scala/com/databricks/spark/xml/util/InferSchema.scala

```diff
@@ -125,15 +125,19 @@ private[xml] object InferSchema {
       datum
     }
 
-    value match {
-      case null => NullType
-      case v if v.isEmpty => NullType
-      case v if isLong(v) => LongType
-      case v if isInteger(v) => IntegerType
-      case v if isDouble(v) => DoubleType
-      case v if isBoolean(v) => BooleanType
-      case v if isTimestamp(v) => TimestampType
-      case _ => StringType
+    if(options.inferSchema){
```

Comment:
Not a big deal, but can we fix style here? spaces around parens.

Reply:
Thanks for the reminder.
I'll fix it.